### PR TITLE
Liked videos

### PR DESCRIPTION
**Pull request by SukanyaSen530**
Page
- Liked Videos Page

Features - 
Logged in user can 
- like a video,
- remove from liked video
- check all liked videos

### CODE REVIEW

File: src/components/Navbar/data.js [hunk 1]
```diff
@@ -15,7 +15,7 @@ export const navData = [
     path: "/watchlater",
     icon: <MdWatchLater />,
   },
-  { id: "n5", name: "Playlist", path: "/playlists", icon: <BsPlayBtnFill /> },
+  // { id: "n5", name: "Playlist", path: "/playlists", icon: <BsPlayBtnFill /> },
```
Comment (AmishMishra11) [inline src/components/Navbar/data.js:18]: remove comments

**Review by AmishMishra11 — APPROVED** [review]
everything else looks good

File: src/components/Navbar/data.js (hunk 1 above)
Comment (SukanyaSen530) [reply src/components/Navbar/data.js:18]: I will need this later when I implement playlist feature :)

Comment (SukanyaSen530) [reply src/components/Navbar/data.js:18]: I will uncomment this when I implement playlist feature,

**Review by sharathnair9999 — APPROVED** [review]
Liked your way of abstracting all possible sensitive info into env file. I'll try to use that approach as well

**Pull request merged**